src/mlte/spec/requirement.py

```diff
@@ -0,0 +1,82 @@
+"""
+Defines a requirement for a measurement to be approved, including the conditions to validate.
+"""
+from __future__ import annotations
+
+from typing import Any
+
+from mlte.validation import Result, Condition
+from mlte.value import Value
+from mlte.evidence import Identifier
+
+# -----------------------------------------------------------------------------
+# Requirement
+# -----------------------------------------------------------------------------
+
+
+class Requirement:
+    """
+    The Requirement class defines a named condition that will be used to validate a property.
+    """
+
+    def __init__(self, identifier: str, condition: Condition) -> None:
+        """Creates a Requirement."""
+        self.identifier = Identifier(identifier)
+        self.condition = condition
+
+    def to_json(self) -> dict[str, Any]:
+        """Returns this requirement as a dictionary."""
+        return {
+            "identifier": str(self.identifier),
+            "condition": self.condition.to_json(),
+        }
+
+    @staticmethod
+    def from_json(document: dict[str, Any]) -> Requirement:
+        """
+        Deserialize a Requirement from a JSON-like dict document.
+
+        :param json: The json document
+        :type json: dict[str, Any]
+
+        :return: The deserialized requirement
+        :rtype: Requirement
+        """
+        if "identifier" not in document or "condition" not in document:
+            raise RuntimeError("Saved requirement is malformed.")
+
+        return Requirement(
+            document["identifier"],
+            Condition.from_json(document["condition"]),
+        )
+
+    def validate(self, value: Value) -> Result:
+        """
+        Validates if the given value matches the requirement.
+
+        :return: The result of validating this requirement.
+        :rtype: Result
+        """
+        result: Result = self.condition(value)
```

Comment:
This is pretty awesome how simple validation becomes now that you have all the machinery in place.